src/tv2-common/showstyle/config-manifests.ts

```diff
@@ -34,16 +45,77 @@ export const getGfxSetupsEntries = (columns: ConfigManifestEntryTable['columns']
 			...columns
 		],
 		hint: ''
-	},
+	}
+]
+
+export const GFX_DEFAULT_VALUES: TableConfigItemGfxDefaults[] = [
 	{
-		id: 'SelectedGfxSetupName',
-		name: 'GFX Setup name',
-		description: 'Name of the GFX Setup that should be used',
-		type: ConfigManifestEntryType.SELECT_FROM_COLUMN,
-		tableId: GRAPHICS_SETUPS_TABLE_ID,
-		columnId: GRAPHICS_SETUPS_NAME_COLUMN_ID,
-		multiple: false,
-		required: false,
-		defaultVal: ''
+		GfxSetup: '',
+		DefaultSchema: '',
+		DefaultDesign: ''
 	}
 ]
+
+export const getGfxDefaults: ConfigManifestEntry = {
+	id: ShowStyleConfigId.GFX_DEFAULTS_TABLE_ID,
+	name: 'GFX Defaults',
+	description: 'The possible defaults available for the GFX setup',
```

Comment:
Possible and available are indicating the same thing.
```suggestion
	description: 'The default values available for the GFX setup',
```